src/constants.ts

```diff
@@ -1,66 +1,74 @@
-import "dotenv";
-
 abstract class Constants {
-	// Status codes
-	static readonly SUCCESS:number = 200;
-	static readonly CREATED:number = 201;
-	static readonly NO_CONTENT:number = 204;
-	static readonly BAD_REQUEST:number = 400;
-	static readonly FAILURE:number = 400;
-	static readonly UNAUTHORIZED_REQUEST:number = 401;
-	static readonly FORBIDDEN:number = 403;
-	static readonly NOT_FOUND:number = 404;
-	static readonly OLD_API:number = 418;
-	static readonly INTERNAL_ERROR:number = 500;
+    // Status codes
+    static readonly SUCCESS: number = 200;
+    static readonly CREATED: number = 201;
+    static readonly NO_CONTENT: number = 204;
+    static readonly BAD_REQUEST: number = 400;
+    static readonly FAILURE: number = 400;
+    static readonly UNAUTHORIZED_REQUEST: number = 401;
+    static readonly FORBIDDEN: number = 403;
+    static readonly NOT_FOUND: number = 404;
+    static readonly OLD_API: number = 418;
+    static readonly INTERNAL_ERROR: number = 500;
+
+    // URLs
+    private static readonly ADMIN_DEVICE: string = "admin";
+    private static readonly ADMIN_REDIRECT: string =
+        "https://admin.hackillinois.org/auth/";
+
+    private static readonly DEV_DEVICE: string = "dev";
+    private static readonly DEV_REDIRECT: string =
+        "https://adonix.hackillinois.org/auth/dev/";
 
-	// URLs
-	private static readonly ADMIN_DEVICE:string = "admin";
-	private static readonly ADMIN_REDIRECT:string = "https://admin.hackillinois.org/auth/";
+    private static readonly WEB_DEVICE: string = "web";
+    private static readonly WEB_REDIRECT: string =
+        "https://www.hackillinois.org/auth/";
 
-	private static readonly DEV_DEVICE:string = "dev";
-	private static readonly DEV_REDIRECT:string = "https://adonix.hackillinois.org/auth/dev/";
+    private static readonly IOS_DEVICE: string = "ios";
+    private static readonly IOS_REDIRECT: string = "hackillinois://login/";
 
-	private static readonly WEB_DEVICE:string = "web";
-	private static readonly WEB_REDIRECT:string = "https://www.hackillinois.org/auth/";
+    private static readonly ANDROID_DEVICE: string = "android";
+    private static readonly ANDROID_REDIRECT: string = "hackillinois://login/";
 
-	private static readonly IOS_DEVICE:string = "ios";
-	private static readonly IOS_REDIRECT:string = "hackillinois://login/";
+    static readonly DEFAULT_DEVICE: string = this.WEB_DEVICE;
+    static readonly DEFAULT_REDIRECT: string = this.WEB_REDIRECT;
 
-	private static readonly ANDROID_DEVICE:string = "android";
-	private static readonly ANDROID_REDIRECT:string = "hackillinois://login/";
-	
-	static readonly DEFAULT_DEVICE:string = this.WEB_DEVICE;
-	static readonly DEFAULT_REDIRECT:string = this.WEB_REDIRECT;
+    static readonly REDIRECT_MAPPINGS: Map<string, string> = new Map<
```

Comment:
@Timothy-Gonzalez  What's happening here?

Reply:
Not the best diff but I think this is just formatting? Will check when I get to a computer.